shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/controler/DatasetProcessingApiController.java

```diff
@@ -112,9 +117,13 @@ public ResponseEntity<List<DatasetDTO>> getOutputDatasets(
 	public ResponseEntity<DatasetProcessingDTO> saveNewDatasetProcessing(
 			@Parameter(description = "dataset processing to create", required = true) @Valid @RequestBody DatasetProcessing datasetProcessing,
 			final BindingResult result) throws RestServiceException {
+
+		/* set authenticated username */
+		datasetProcessing.setUsername(KeycloakUtil.getTokenUserName());
```

Comment:
When coming back from VIP execution, it's correctly set ? (as it's an external running thread)

Reply:
Ok